docs/conf.py

```diff
@@ -1,97 +0,0 @@
-# Configuration file for the Sphinx documentation builder.
```

Comment:
Need to review the config and docs in general the assert it works as expected after changing the framework. Specially the buttons to launch on binder.